### PR TITLE
Extended documentation for Grafana dashboards.

### PR DESCRIPTION
**Pull request by cosmic-matter**

### CODE REVIEW

**Review by Yesoufatiy — CHANGES_REQUESTED** [review]
Maybe you can add the Table of Contents and Links to the documentation now?

**Comment by dannysongg** [comment]
The extra detail in the grafana setup is a welcome addition. Looks like it'll make that process much easier in the future!

File: monitoring/README.md [hunk 1]
```diff
@@ -169,11 +169,17 @@ At this point, all three services should be set up: the monitoring server on the
 
 ## Making the dashboard
 
-1. Create a new dashboard by clicking on the "plus" symbol in the menu to the left, and in the dropdown which appears, "Dashboard".
+1. Create a new dashboard by clicking on the "plus" symbol in the menu to the left, and in the dropdown which appears, "Dashboard". 
+
+2. In the new dashboard, click "Add new panel". A panel will allow you to organize the monitoring data. This will open a configuration page with many options. At the top you can edit the name of the panel. The main option to look for is within the first tab under the graph which appears, called "Query". Here is where you will specify which timeserie you would like to display on the panel. You simply click on 'select metric' and select a piece of data from one of the pis on the drop down menu. Each query displays only one statistic for each pi, so naturally you will need to add more queries with the '+ query' button to display the same statistic but for the other pis.
```
Comment (breed) [inline monitoring/README.md]: timeserie -> timeseries.

does this have to be one line? i think you can have a sentence on each line as long as you indent the start of the sentence.

File: monitoring/README.md [hunk 2]
```diff
@@ -157,7 +157,7 @@ At this point, all three services should be set up: the monitoring server on the
 
 1. Follow [these instructions]() to install and run Grafana on the VM. The instructions are for Debian/Ubuntu, since this is the OS which the VM runs.
 
-2. To access the Grafana dashboard from your local computer, make sure you are connected to the VPN for the switch (you should already have openvpn set up by this point). Open a browser and navigate to your switch address: for example, `172.31.x.254:3000`, where x is your group number. The port is 3000 since this is Grafana's default server port. 
+2. To access the Grafana dashboard from your local computer, make sure you are connected to the VPN for the switch (you should already have openvpn set up by this point). Open a browser and navigate to your switch address: for example, `172.31.x.254:3000`, where x is your group number. The port is 3000 since this is Grafana's default server port.
```
Comment (breed) [inline monitoring/README.md]: it's not your switch address. it's the piman address right?

**Review by breed — CHANGES_REQUESTED** [review]

File: monitoring/README.md (hunk 1 above)
Comment (cosmic-matter) [reply monitoring/README.md]: Ah gotcha. It is quite bulky. I can break it down.

**Review by Yesoufatiy — APPROVED** [review]
Contents all display better now

File: monitoring/README.md (hunk 2 above)
Comment (cosmic-matter) [reply monitoring/README.md]: Oh right I made a change to that

File: monitoring/README.md [hunk 3]
```diff
@@ -171,9 +186,16 @@ At this point, all three services should be set up: the monitoring server on the
...
+  At the top you can edit the name of the panel.  
+  The main option to look for is within the first tab under the graph which appears, called "Query". Here is where you will specify which timeseries you would like to display on the panel.  
+  Simply click on 'select metric' and select a piece of data from one of the pis on the drop down menu.  
+  Each query displays only one statistic for each pi, so naturally you will need to add more queries with the '+ query' button to display the same statistic but for the other pis.  
+
+3. On the right side, under 'visualization' there are options to modify how the data is viewed. The options include using a graph or a gauge. By default, a graph is chosen. The 'display' option underneath 'visualization' shows how each query is shown, and the default is a line.
+
+4.  Directly on top of the graph there is a clock icon with a drop down menu where you can select the time frame that you want the data to be shown in. The icon with the down arrow on the farthest right gives the options of when to refresh the data.
```
Comment (breed) [inline monitoring/README.md]: you have two leading spaces instead of one

**Review by breed — CHANGES_REQUESTED** [review]
almost there. one small change.

**Review by Yesoufatiy — APPROVED** [review]

**Review by breed — APPROVED** [review]

**Pull request merged**